lexicons/app/bsky/feed/generator.json

```diff
@@ -39,6 +39,12 @@
             "description": "Self-label values",
             "refs": ["com.atproto.label.defs#selfLabels"]
           },
+          "langs": {
+            "type": "array",
+            "description": "Indicates the human language for the intended audiences of the feed generator.",
```

Comment:
Should we mention the format here? Like pry leave it open, but specify at least BCP-47 with or without region codes?